eZ/Publish/API/Repository/Values/User/RoleCreateStruct.php

```diff
@@ -18,6 +18,13 @@
 abstract class RoleCreateStruct extends ValueObject
 {
     /**
+     * ID of a role.
+     *
+     * @var int
+     */
+    public $id;
```

Comment:
It's introducing an inconsistency with other structs and other repository services. AFAICT, we don't have any other structs holding an existing ID.